core/src/main/java/com/exadel/aem/toolkit/core/optionprovider/services/impl/HttpOptionSourceResolver.java

```diff
@@ -78,9 +78,27 @@ class HttpOptionSourceResolver implements OptionSourceResolver {
      * {@inheritDoc}
      */
     @Override
-    public Resource resolve(SlingHttpServletRequest request, String uri) {
-        String internalPath = getInternalPath(uri);
-        String url = StringUtils.removeEnd(uri, internalPath);
+    public Resource pathResolve(SlingHttpServletRequest request, PathParameters pathParameters) {
+        return resolve(request, pathParameters.getPath());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Resource fallbackResolve(SlingHttpServletRequest request, PathParameters pathParameters) {
+        return resolve(request, pathParameters.getFallbackPath());
+    }
+
+    /**
+     * Resolves
```

Comment:
Could you please add more details to description of this method? what exactly this method resolves?